contrib/opencensus-ext-ocagent/opencensus/ext/ocagent/utils/__init__.py

```diff
@@ -59,6 +58,6 @@ def proto_ts_from_datetime_str(dt):
     if (dt is not None):
         try:
             ts.FromJsonString(dt)
-        except ParseError:
+        except ValueError:
```

Comment:
This might have backward-compat issue if people are specifying old version (e.g. using dependencies.txt file) of protocol buffer lib.

```python
from google.protobuf.internal.well_known_types import ParseError
err = ParseError()
print(isinstance(err, ValueError))  # gives False
```

Reply:
That's true, I added a note in CHANGELOG. With the new protobuf release this change is inevitable though.